setup.py

```diff
@@ -22,10 +22,7 @@
         'pytest>=2.9,<3.0',
         'six',
     ],
-    packages=[
-        'dynamallow',
-    ],
-
+    packages=find_packages('.', exclude=['tests']),
```

Comment:
If we're going this route let's also exclude `docs` since there is some Python code in there.